runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -565,12 +580,17 @@ Color getHighlighted(String item, int gePrice, int haPrice)
 			return config.highlightedColor();
 		}
 
-		// Explicit hide takes priority over implicit highlight
+		// Explicit hide takes priority over implicit highlight and inventory stacks
 		if (TRUE.equals(hiddenItems.getUnchecked(item)))
 		{
 			return null;
 		}
 
+		if (config.inventoryStacks() && inventoryStacks != null &&  TRUE.equals(inventoryStacks.getUnchecked(item)))
```

Comment:
There's a double space in this `if` statement.

```suggestion
		if (config.inventoryStacks() && inventoryStacks != null && TRUE.equals(inventoryStacks.getUnchecked(item)))
```